include/micm/solver/cuda_linear_solver.cuh

```diff
@@ -4,6 +4,9 @@
  */
 #pragma once
 
+
+#include <chrono>
```

Comment:
there is no timing measurement in this file so I don't think we need this header file?

Reply:
Nope!  Removed!